cypress.config.ts

```diff
@@ -32,6 +32,7 @@ export default defineConfig({
           LOGIN_USER: process.env.CYPRESS_LOGIN_USER,
           LOGIN_PASSWORD: process.env.CYPRESS_LOGIN_PASSWORD,
           PROJECT_KEY: process.env.CYPRESS_PROJECT_KEY,
+          PACKAGE_NAME: process.env.CYPRESS_PACKAGE_NAME,
```

Comment:
As suggested by Carlos, I moved reading this env variable from the command:
```
pnpm start-server-and-test 'pnpm template-${{ inputs.application-type }}-${{ inputs.template-name }}:start:prod:local' http-get://127.0.0.1:3001 'pnpm ${{ inputs.e2e-tests-command }} --env PACKAGE_NAME=${{ inputs.package-name }}'
```
to our Cypress config